### ISSUE TITLE
New comments of private discussion in stream

### ISSUE THREAD

**Issue by lordyavin** [theCrag/website#2854]
**What happened?**
I see the new comments of  "***Dangerous and anti-social behaviour at Upper Slider wall on Saturday***" discussion in my stream but I it is private now. 

**What you expected:**
I should not see any new comments of a private discussion in my stream.

**Comment by scd** [comment]
Unlinking removed it from the node but not the stream

**Comment by brendanheywood** [comment]
There is still the broader issue of why Kai ever saw updates to this in the first place

**Comment by scd** [comment]
I think because you commented and he is following you. #2853

**Comment by brendanheywood** [comment]
Yeah I know that's why, I'm saying the business rules are wrong. He should only see stream events which directly involve me. He should not see stream updates to a thread which I am subscribed to but he is not. He should not ever need to unwatch it because he should never have been watching it in the first place.
